src/pages/Loans/LoansOverview/components/LoanForm/LoanForm.tsx

```diff
@@ -128,20 +128,33 @@ const LoanForm = ({ asset, variant, position, onChangeLoan }: LoanFormProps): JS
     data: { hasCollateral }
   } = useGetAccountPositions();
   const prices = useGetPrices();
-  const { governanceBalance, assetAmount, assetPrice, transactionFee } = useLoanFormData(variant, asset, position);
+  const { governanceBalance, assetAmount, assetPrice, transactionFee } = useLoanFormData(
+    variant,
```

Comment:
This is a bit confusing—why are there two `variant` arguments?

Reply:
Good point! I will fix that. Thank you